x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/advanced_step/advanced_step_form.tsx

```diff
@@ -61,6 +78,40 @@ export const AdvancedStepForm: FC<CreateAnalyticsStepProps> = ({
 
   const mmlInvalid = modelMemoryLimitValidationResult !== null;
 
+  const isStepInvalid = mmlInvalid || Object.keys(advancedParamErrors).length > 0;
+
+  useEffect(() => {
+    (async function () {
+      const { success, errorMessage } = await getExplainData(form);
+      const paramErrors: {
+        [key in ANALYSIS_ADVANCED_FIELDS]?: string;
+      } = {};
+
+      if (!success) {
+        // Check which field is invalid
+        Object.values(ANALYSIS_ADVANCED_FIELDS).forEach((param) => {
+          if (errorMessage.includes(`[${param}]`)) {
+            paramErrors[param] = errorMessage;
+          }
+        });
+      }
+      setAdvancedParamErrors(paramErrors);
```

Comment:
Since this code relies on remote data to set the errors, I wonder if we should disable the wizard button while the remote call is ongoing (thinking of a slow connection where a user entered an update, validation wasn't finished and user continued to the next step).

Reply:
Good call - updated in 8837c39c1d075444f078c6d82434a820b8e2dbe3